deploy/determined_deploy/aws/templates/secure.yaml

```diff
@@ -1,5 +1,15 @@
 Description:  This template deploys a VPC, with a public and private subnet. It deploys an internet gateway,
-  with a default route on the public subnet. It a NAT gateway, and default routes for them in the private subnet.
+  with a default route on the public subnet with a NAT gateway, and default routes for them in the private subnet.
+Mappings:
+  RegionMap:
+    us-east-1:
```

Comment:
Eventually we might want to consider decoupling the template and `det-deploy` supported region checks.  The general idea would be:
- we only have the environments image configured in the CF templates and det-deploy. maybe the Owner and AMI Name, or just the AMI Source, which in our case is `<Owner>/<AMI Name>`.  The CF templates and det-deploy checks would dynamically look up AMI IDs.

I'm not sure how smooth it would be to do this sort of dynamic lookup in CF though.

If we move toward dynamic AMI ID lookup, then region support changes and environments image updates would get a lot easier -- if we add another supported region, all you have to do is update the configured Owner / Source / AMI Name here.  Similarly if we update our environments image, you just have one line to update here rather than AMI IDs for all supported regions

Reply:
It's something we could get to eventually. It gets a little hairy because you need to use a custom resource (basically build and execute a lambda) to fetch the AMI. I don't think we've solidified our environments publishing to make it a smooth right now.